assets/js/universal-dapp.js

```diff
@@ -255,6 +255,19 @@ UniversalDApp.prototype.getCallButton = function(args) {
         return $gasUsed;
     }
 
+    var getDecodedOutput = function (result) {
+        var $decoded;
+        if (Array.isArray(result)) {
+            $decoded = $('<ol>');
+            for (var i = 0; i < result.length; i++) {
+                $decoded.append($('<li>').html(result[i]));
+            }
+        } else {
+            $decoded = result;
+        }
+        return $('<div class="decoded">').html('<strong>Decoded:</strong> ').append($decoded);
```

Comment:
If you look at the output I think it makes sense having the `Decoded:` or `Parsed:` in front, because the data is presented as a list. See examples in the first message in this PR.